README.md

```diff
@@ -1,5 +1,5 @@
 # Google Student Training in Engineering Program
 
-This repo contains Lujie "Sue" Zhao's portfolio and STEP projects.
+This repo contains Sue's portfolio and STEP starter projects.
 
-Description related to the intern program can be found [here] (https://buildyourfuture.withgoogle.com/programs/step/)
+Description related to the intern program can be found [here](https://buildyourfuture.withgoogle.com/programs/step/)
```

Comment:
I thinks I miss this in #3 but there should be a `.` at the end.

Reply:
I just added the period!